lms/templates/vert_module.html

```diff
@@ -27,7 +27,7 @@ <h2 class="hd hd-2 unit-title">${unit_title}</h2>
     % if completed:
       <div class="pill completed">${_('Completed')}</div>
     % elif past_due:
-      <div class="pill past-due">${_('Past Due')}</div>
+      <div class="pill past-due">${_('Past due')}</div>
```

Comment:
intentional?

Reply:
Yeah, the lowercase will match the newer design used in the MFE.